kchd/hardware.py

```diff
@@ -10,16 +10,16 @@ class KCHLED(enum.IntEnum):
     The value of each LED in the Enum is the BCM pin number.
     """
 
-    BOOT_20 = 7
-    BOOT_40 = 5
+    # BOOT_20 = 7  # Controlled by hat EEPROM GPIO map
+    # BOOT_40 = 5  # Controlled by systemd
```

Comment:
We need to keep this as KCH intentionally takes over these LEDs.

When the updating animation is implemented, all of the boot LEDs will be used.